### PR TITLE
Fix download link, remove jQuery, small improvements

### PR DESCRIPTION
**Pull request by rkuykendall**
## Changes

- Removed jQuery and replace the two functions we used. [getElementsByClassName](https://caniuse.com/getelementsbyclassname) and [DOMContentLoaded](https://caniuse.com/?search=DOMContentLoaded) are both supported in IE9+.
- Cleaned up meta tags to match modern standards
- Optimized banner for faster First Paint
- Changed fonts to swap for faster First Paint
- Fixed some outdated S7 links
- Fixed broken `{{ dl_alephflatpak }}` link
- Added viewport for first baby-step towards mobile

## Chrome Lighthouse before / after
Screenshot 2023-08-04 at 12 13 41 PM
Screenshot 2023-08-04 at 12 13 43 PM

Screenshot 2023-08-04 at 12 12 22 PM
Screenshot 2023-08-04 at 12 12 24 PM

### CODE REVIEW

**Comment by treellama** [comment]
These are good. I had the flatpak download link going to the download page instead of a direct download because we've seen lots of newbies with Linux and I think they could use the directions on the click through page.

I'd rather not set a viewport for now--this site is explicitly for desktops.

File: _includes/header.html [hunk 1]
```diff
@@ -1,17 +1,16 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
-                      "http://www.w3.org/TR/html4/loose.dtd">
-<html>
+<!DOCTYPE HTML>
+<html lang="en">
 {% capture dummy %}{% include version_info.md %}{% endcapture %}
   <head>
     <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (rkuykendall) [inline _includes/header.html:6]:
```suggestion
```

**Comment by rkuykendall** [comment]
@treellama Removed viewport. Updated Lighthouse below:

image

Do you not want the site to be made mobile-friendly at all, or just want it to be all in one PR?

**Comment by treellama** [comment]
At the moment I don't have any plans for a mobile site, since the site is for desktop downloads.

**Pull request merged**